jsx/StaticDataTable.js

```diff
@@ -264,21 +264,19 @@ var StaticDataTable = React.createClass({
     var index = [];
     var that = this;
 
-    for (var i = 0; i < this.props.Data.length; i += 1) {
-      var val = this.props.Data[i][this.state.SortColumn];
-
-      if (parseInt(val, 10) === val) {
-        val = parseInt(val, 10);
-      } else if (parseFloat(val) === val) {
-        val = parseFloat(val);
-      } else if (val === '.') {
```

Comment:
Is it complicated to update DQT to use `undefined` or `null` to check for non-existent? The `.` seems kind of misleading